tensorflow_addons/custom_ops/image/cc/kernels/image_projective_transform_op.cc

```diff
@@ -43,32 +43,70 @@ template struct FillProjectiveTransform<CPUDevice, double>;
 
 }  // end namespace functor
 
-typedef Eigen::ThreadPoolDevice CPUDevice;
+using CPUDevice = Eigen::ThreadPoolDevice;
```

Comment:
Are we forcing to run all these ops on CPU only?

Reply:
Nope, GPUDevice is also registered [here](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/custom_ops/image/cc/kernels/image_projective_transform_op.cc#L148-L160).

Reply:
Cool. Thanks for the clarification